core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/DrmSessionErrorForwarder.java

```diff
@@ -1,38 +0,0 @@
-package com.novoda.noplayer.internal.exoplayer.forwarder;
-
-import com.google.android.exoplayer2.drm.DefaultDrmSessionManager;
-import com.novoda.noplayer.NoPlayerError;
-import com.novoda.noplayer.NoPlayer;
-import com.novoda.noplayer.PlayerErrorType;
-
-import static com.novoda.noplayer.internal.exoplayer.forwarder.ErrorFormatter.formatMessage;
-
-class DrmSessionErrorForwarder implements DefaultDrmSessionManager.EventListener {
-
-    private final NoPlayer.ErrorListener errorListener;
-
-    DrmSessionErrorForwarder(NoPlayer.ErrorListener errorListener) {
-        this.errorListener = errorListener;
-    }
-
-    @Override
-    public void onDrmKeysLoaded() {
-        // TODO: should we send?
-    }
-
-    @Override
-    public void onDrmSessionManagerError(Exception e) {
```

Comment:
According to the documentation:

```
    /**
     * Called when a drm error occurs.
     * <p>
     * This method being called does not indicate that playback has failed, or that it will fail.
     * The player may be able to recover from the error and continue. Hence applications should
     * <em>not</em> implement this method to display a user visible error or initiate an application
     * level retry ({@link Player.EventListener#onPlayerError} is the appropriate place to implement
     * such behavior). This method is called to provide the application with an opportunity to log
     * the error if it wishes to do so.
     *
     * @param e The corresponding exception.
     */
```

So we shouldn't be delegating to `PlayerError` for client handling and instead of that simply logging